securitycenter/management_api/event_threat_detection/event_threat_detection_custom_module_test.go

```diff
@@ -225,10 +220,13 @@ func TestCreateEtdCustomModule(t *testing.T) {
 func TestGetEtdCustomModule(t *testing.T) {
 	var buf bytes.Buffer
 
+	mu.Lock()
```

Comment:
question: why the mutex here? What shared resource is being protected inside the critical section / this function?